test cases/common/125 shared module/module.c

```diff
@@ -9,54 +9,6 @@
   #endif
 #endif
 
-#ifdef _WIN32
```

Comment:
No, I mean linking, not linking to: using this test case as an example of `shared_module` could give the impression that you are never meant to write `shared_module(..., link_with: otherlib)`